docs/about.md

```diff
@@ -34,3 +34,21 @@ Talawa’s desired main features are:
 1. Donation acceptance
 1. Service provider level multi-organization capability
 1. Plugin support for administration services such as billing
+
+### Technologies Used
+
+Technology plays an increasingly significant role in improving access to education for people living in impoverished areas and developing countries. Some of the technologies that are being used by Talawa are:
+
+1. Front-end Technologies: HTML, CSS, and JavaScript (React) is used to create the website's front-end user interface. And, Flutter is used to create the mobile application. 
+
+2. Back-end Technologies: Talawa uses server-side scripting languages like GraphQL and TypeScript to handle the website's back-end functionality.
+
+3. Database Management System: Talawa API uses database management system like MongoDB with the mongoose ORM (Object–relational mapping) which is used to store and manage the website's data.
+
+4. Security: Talawa uses various encryption technologies to protect sensitive data like SSL (Secure Socket Layer) which helps to create a secure encrypted connection between the server(our project) and the client and protects Talawa from hacking and other security threats.
```

Comment:
Use ( 'space' and 'comma' ) between " `server (our project) and the client, and protects Talawa...` "

Reply:
Noted! @salman-farooqi